scout/server/blueprints/cases/templates/cases/causatives.html

```diff
@@ -146,4 +129,23 @@ <h5 class="mb-1">{{ variant.hgnc_symbols|join(', ') }}</h5>
       });
     })
   </script>
+  <script language="JavaScript">
+    $(document).ready(function () {
+      var table = $('#causatives_table').DataTable({
+        lengthChange: false,
+        buttons: ['copy', 'excel']
+      });
+
+      table.buttons().container()
+
+        .appendTo("#causatives_table_wrapper .col-md-6:eq(0)");
+    });
+  </script>
+  <script src="https://cdn.datatables.net/1.10.16/js/jquery.dataTables.min.js"></script>
```

Comment:
Honestly I think its a bit hard to have contol over layout etc when using all the libraries. But since they make life easier over all I think its worth it. Regarding the orientation of the search field eg, I dont understand why it ends up where it does. In other code I have made with datatables, i did not get that issue. I tried to figure out what is causing it, without progress. I now solved it by adding "dom: 'Bf'" to the datatable. Im not sure why it fixed it though. Would be nice to have better understanding of what is happening there.